src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(result, value) {
```

Comment:
The add, subtract, multiply, and divide methods shouldn't take result as a parameter. They should operate on this.result

Reply:
I don't know why but if I try using this.result directly into methods, I can't pass tests...

Reply:
Try to use like this
```
add(value) {
  this.result += value;

  return this;
},
```

Reply:
It's not working too(